OrbitCore/ContextSwitch.h

```diff
@@ -11,9 +13,11 @@ struct ContextSwitch
     ContextSwitch( SwitchType a_Type = Invalid );
     ~ContextSwitch();
 
```

Comment:
Needed to use uintN_t types here, as otherwise serialization will fail on Windows.